src/macaron/config/defaults.ini

```diff
@@ -60,6 +60,7 @@ redirect_urls =
     gitbox.apache.org
     git-wip-us.apache.org
 find_source_should_clone = False
+try_latest_purl = True
```

Comment:
Can you please add a comment to explain the option (and while at it for `find_source_should_clone` as well)?